modules/nw-network-flows-create.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/tracking-network-flows.adoc
+
+[id="nw-network-flows-create_{context}"]
+= Adding destinations for network flows collectors
+
+As a cluster administator, you can configure the Cluster Network Operator to send network flows metadata about the pod network to a network flows collector.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+* You have a network flows collector and know the IP address and port that it listens on.
+
+.Procedure
+
+. Create a manifest file that specifies the network flows collector type and the IP address and port information of the collectors.
+
+. Run the following command to configure the Cluster Network Operator with the network flows collectors:
++
+[source,terminal]
+----
+$ oc apply -f <file-name>.yaml
+----
++
+.Example output
+[source,terminal]
+----
+network.operator.openshift.io/cluster configured
+----
+
+.Verification
+
+FIXME
+
+. We don't want to show the foll because that's what is requested, not necessarily what's running:
++
+[source,terminal]
+----
+$ k get network.operator cluster -o jsonpath='{.spec.exportNetworkFlows}'
+----
+
+. The torturous path is to `oc debug node/<node-name>`, chroot, and then run the following, but that seems a little much:
++
+[source,terminal]
+----
+# ovsdb-tool -mm show-log
+----
++
+.Example output
+[source,terminal]
+----
+record 101: 2021-04-30 15:53:41.939 "ovs-vsctl (invoked by /usr/bin/ovnkube): /usr/bin/ovs-vsctl --timeout=15 -- --id=@netflow create netflow "targets=[\"17
+2.30.198.159:2056\"]" active_timeout=60 -- set bridge br-int netflow=@netflow"
+  table Bridge row "br-int" (2951d5eb) diff:
+    netflow=58d0ad55-c79e-4dbb-870a-76ed8835e82e
+  table NetFlow row b0ee1bbe (b0ee1bbe) diff:
+    delete row
+  table NetFlow insert row 58d0ad55:
+    active_timeout=60
+    targets=["172.30.198.159:2056"]
+  table Open_vSwitch row 9d85c043 (9d85c043) diff:
+    next_cfg=30
+----
+
+. What is the real way?
```

Comment:
one way

```
for f in $(oc get pods -n openshift-ovn-kubernetes -l app=ovnkube-node -o jsonpath='{range@.items[*]}{.metadata.name}{"\n"}{end}' ) ; do oc  -n openshift-ovn-kubernetes exec -c ovnkube-node  $f -- bash -c 'for f in ipfix sflow netflow ; do  ovs-vsctl find $f ; done'   ; done
```

could maybe use `-f table` if the terminal is wide enough

```
_uuid               : 17f505e3-2980-44c9-ab0b-04a204bab450
cache_active_timeout: 60
cache_max_flows     : []
external_ids        : {}
obs_domain_id       : []
obs_point_id        : []
other_config        : {}
sampling            : []
targets             : ["10.128.2.12:2055", "10.128.2.13:2055", "10.129.2.7:2055", "10.129.2.8:2055"]
_uuid               : 8aff65aa-3cff-4c06-af83-76d01c545316
agent               : ovn-k8s-mp0
external_ids        : {}
header              : []
polling             : []
sampling            : []
targets             : ["10.128.2.14:6343", "10.129.2.9:6343", "10.131.0.7:6343"]
_uuid               : 08a0dae0-5068-4a22-9038-029855d88e5f
active_timeout      : 60
add_id_to_interface : false
engine_id           : []
engine_type         : []
external_ids        : {}
targets             : ["10.128.2.15:2056", "10.131.0.8:2056", "10.131.0.9:2056"]
```

Reply:
Sweet.  I started with your work and made it a little prettier--please let me know if you can check it too.

Also, I only showed a single collector type:
A) We're not all over achievers.
B) Small guess that an enterprise picks a single record type for standardization and to concentrate efforts.

This is a huge help.  Thank you.